src/index.js

```diff
@@ -0,0 +1,5 @@
+const firstNumber = +prompt('Enter your first number');
+const secondNumber = +prompt('Enter your second number');
+const sum = firstNumber + secondNumber;
+
+alert(`Your first number is ${firstNumber}, your second·number is ${secondNumber}. Result of adding these two numbers is ${sum}.`)
```

Comment:
Change format to 
alert(
  `Your first number is ${firstNumber}, your second·number is ${secondNumber}. Result of adding these two numbers is ${sum}.`,
);